nwbinspector/checks/time_series.py

```diff
@@ -46,7 +46,11 @@ def check_data_orientation(time_series: TimeSeries):
 
 @register_check(importance=Importance.CRITICAL, neurodata_type=TimeSeries)
 def check_timestamps_match_first_dimension(time_series: TimeSeries):
-    """If the TimeSeries has timestamps, check if their length is the same as the zero-axis of data."""
+    """
+    If the TimeSeries has timestamps, check if their length is the same as the zero-axis of data.
+
+    Best Practice: :ref:`best_practice_data_orientation`
+    """
```

Comment:
Note that this is what we have to do in those cases where a Best Practice has multiple ways of checking for violations.